src/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.Escaping.cs

```diff
@@ -5,16 +5,9 @@
 using System.Buffers;
 using System.Buffers.Text;
 using System.Diagnostics;
-using System.Numerics;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;  // Do not remove. Needed for Int32LsbToHexDigit when !BUILDING_INBOX_LIBRARY
```

Comment:
Good idea. I will do this separately (for master) since it is primarily a cosmetic change and I want to cherry-pick this change for 3.1 (was waiting on CI to be green for a while :p).